frontend/src/js/common-ui/forms/form.js

```diff
@@ -41,6 +41,12 @@ const getErrorMsg = (validateMethod, args) => {
         return `Must be at least ${args[0]} characters long`;
       }
       break;
+    case 'isAlphanumericLocator':
+      if (args[0] && validator.isWhitelisted(args[0], 'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789_-')) {
```

Comment:
maybe 
```suggestion
      if (args[0] && validator.matches(args[0], /^[a-zA-Z0-9_-]+$/)) {
```
is a better way?